### DISCUSSION TITLE
Corrupted checkpoint

### DISCUSSION THREAD

**Discussion by Linardos** [FETS-AI/Challenge#94] [Q&A] [answered]
My process is getting killed way too often for some reason (even though the GPU usage is barely above 60% of its total when I check it at nvidia-smi). It's just a baseline run of partitioning_1.csv with fixed_Batch_size, all_collaborators, simple weight average. This time it happened to get killed during storage and resulted in a corrupted pickle file when I ran it again:
![image](https://user-images.githubusercontent.com/26694607/125608915-8b09b611-2514-426b-bbe0-054ca92c9f83.png)
Is there a way to store multiple intermediate checkpoints to avoid losing the entire experiment in case this happens again?

**Comment by sarthakpati** [comment, marked as answer]
You could potentially edit the [`checkpoint_utils`](https://github.com/FETS-AI/Challenge/blob/main/Task_1/fets_challenge/checkpoint_utils.py) module to add more locations for saving. Be warned that none of these customizations can be part of the submission.
